lit_gpt/lora.py

```diff
@@ -62,7 +62,7 @@
 
 
 class LoRALayer:
-    def __init__(self, r: int, lora_alpha: int, lora_dropout: float, merge_weights: bool):
```

Comment:
For reference, this change was discussed in https://github.com/Lightning-AI/lit-gpt/pull/296#discussion_r1271297881

The arguments here are no longer needed, since we decided to make the merging a functional call instead of doing it through eval()/train() calls. The argument no longer does anything meaningful, so it can be removed.

Reply:
You are talking about `merge_weights` argument, right?
If so - I believe it was removed in the PR (red color) :)

Reply:
I left this comment to provide context to the other reviewers (@carmocca) because the two of us initially talked about this in a different PR.

Reply:
Oh, ok :)